softgroup/basis/common-jwt/src/main/java/com/softgroup/common/jwt/api/TokenService.java

```diff
@@ -0,0 +1,31 @@
+package com.softgroup.common.jwt.api;
+
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.MalformedJwtException;
+import io.jsonwebtoken.SignatureException;
+
+/**
+ * @author AlexKazmirchuk
+ * @since 16.03.17.
+ */
+
+public interface TokenService {
+
+    void validateSessionToken(String sessionToken) throws ExpiredJwtException, MalformedJwtException, SignatureException;
```

Comment:
@Dominys  for example instead " ExpiredJwtException, MalformedJwtException, SignatureException"

I must throw my custom "InvalidTokenException" ?  

or method signature must be something like : " ExpiredJwtException, MalformedJwtException, SignatureException, InvalidTokenException" ?